static/js/pm_list.js

```diff
@@ -26,8 +26,8 @@ function update_count_in_dom(count_span, value_span, count) {
     }
 }
 
-function set_count(type, name, count) {
-    var count_span = get_filter_li(type, name).find('.count');
```

Comment:
This looks like a logic error: `get_filter_li` does not take any arguments, so `type` and `name` are ignored here.